### PR TITLE
Feature/donation link qr code

### PR DESCRIPTION
**Pull request by Shreyaschorge**
#Feature
- Add QR Code for donation link at route `/profile/donation-link`
- QR Code will update as per user's input

### CODE REVIEW

**Comment by mariahosfeld** [comment]
Quick first check looks good, but also a download or copy function for the QR-Code needs to be included.

**Comment by mariahosfeld** [comment]
The functionality works for me.
As the explanation for the QR-Code is now at the top, please change the "this" to "the".

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/\[p\]**](https://planet-webapp-gfo2nqq5u-planetapp.vercel.app/[p]) |
|:-:|
|<a href="https://planet-webapp-gfo2nqq5u-planetapp.vercel.app/[p]">Screenshot of /[p]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/v_mo8f5gXktVy0kt_XF_XE1v1WMZGH2KL2v6xgr3mgq1wZJMebHEtj58OkFBZmR1F7Fr4MKtxhrtMnJY5J-vbg.png">(view full size)</a> |







Commit 299efd74bec8d032aaa000895be5d92627250ade (https://planet-webapp-gfo2nqq5u-planetapp.vercel.app).

**Review by norbertschuler — APPROVED** [review]
LGFM

File: package.json [hunk 1]
```diff
@@ -100,7 +102,7 @@
   "devDependencies": {
     "@emotion/babel-plugin": "^11.9.2",
     "@emotion/eslint-plugin": "^11.7.0",
-    "@netlify/plugin-nextjs": "^4.28.6",
+    "@netlify/plugin-nextjs": "^4.30.0",
```
Comment (norbertschuler) [inline package.json:105]: Not sure why this `@netlify/plugin-nextjs` update has to be part of this PR, but otherwise it is still ok.

Comment (Shreyaschorge) [reply package.json:105]: Build was failing https://github.com/Plant-for-the-Planet-org/planet-webapp/actions/runs/3927599948/jobs/6715360301

File: src/features/user/Widget/DonationLink/DonationLinkForm.tsx [hunk 2]
```diff
@@ -227,24 +250,47 @@ const DonationLinkForm = ({
...
-              isVisible={isLinkUpdated}
-              handleClose={handleSnackbarClose}
-            />
+          {qrCode && (
+            <div className={styles.formSection}>
+              <div className={styles.formHeader}>
+                {t('donationLink:qrCodeTitle')}
+              </div>
+              <img
+                style={{ display: 'block', width: '150px', height: '150px' }}
```
Comment (sunilsabatp) [inline src/features/user/Widget/DonationLink/DonationLinkForm.tsx:270]: try to avoid inline styling

**Pull request merged**